js/ui/watermark.js

```diff
@@ -216,6 +216,9 @@ const WatermarkManager = new Lang.Class({
 });
 
 function _forEachBackgroundManager(func) {
-    Main.overview._bgManagers.forEach(func);
-    Main.layoutManager._bgManagers.forEach(func);
+    if (Main.overview._bgManagers != null)
```

Comment:
You can omit the `!= null`, so that the code will not risk running when `Main.overview._bgManagers` is e.g. `undefined`.